Trait bound error caused by diff windows-rs dependence version

windows-rs version in ferrisetw's Cargo.toml is 0.39, but windows-rs version in my project is 0.42. This will cause the following error when compiling:

```
error[E0277]: the trait bound `Parser<'_>: TryParse<GUID>` is not satisfied
  --> src/main.rs:23:66
   |
23 |                     let guid: GUID = TryParse::<GUID>::try_parse(&mut parser, "Guid").unwrap();
   |                                      --------------------------- ^^^^^^^^^^^ the trait `TryParse<GUID>` is not implemented for `Parser<'_>`
   |                                      |
   |                                      required by a bound introduced by this call
   |
   = help: the following other types implement trait `TryParse<T>`:
```

My project can only be compiled by using windows-rs with the same version as ferrisetw.

Is there any good solution?

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -120,3 +120,11 @@ pub mod schema;
 pub mod trace;
 mod traits;
 mod utils;
+
+// These types are returned by some public APIs of this crate.
+// They must be re-exported, so that users of the crate have a way to avoid version conflicts
+// (see https://github.com/n4r1b/ferrisetw/issues/46)
+/// Re-exported `GUID` from `windows-rs`, which is used in return values for some functions of this crate
+pub use windows::core::GUID;
+/// Re-exported `SID` from `windows-rs`, which is used in return values for some functions of this crate
+pub use windows::Win32::Security::SID;

